src/AnalyticsValues.php

```diff
@@ -5,10 +5,15 @@
  * @package P4MT
  */
 
+namespace P4\MasterTheme;
+
+use P4\MasterTheme\Smartsheet;
```

Comment:
I might remove these internal imports again as they're not really needed. They were added by PHPStorm as I renamed the classes one by one, and each time it needed to add it first to classes that used the renamed class but were not in the namespace yet.

Reply:
done